Source/MonoGame.Extended/Maps/Tiled/TiledObject.cs

```diff
@@ -1,18 +1,26 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using Microsoft.Xna.Framework;
 using MonoGame.Extended.Shapes;
 
 namespace MonoGame.Extended.Maps.Tiled
 {
     public class TiledObject
     {
```

Comment:
Rather than having 2 public constructors and a private one you can simply have 2 public constructors and call one from the other using `this()`